docs/resources/v2-conformance-statement.md

```diff
@@ -0,0 +1,818 @@
+| ModalitiesInStudy | X | X | X |  |  |  |
+| SeriesInstanceUID |  | X | X | X | X |  |
+| Modality |  | X | X | X | X |  |
+| PerformedProcedureStepStartDate |  | X | X | X | X |  |
+| ManufacturerModelName | | X | X | X | X |  |
+| SOPInstanceUID |  |  | X |  | X | X |
+
+#### Search Matching
+
+We support below matching types.
+
+| Search Type | Supported Attribute | Example |
+| :---------- | :------------------ | :------ |
+| Range Query | StudyDate, PatientBirthDate | {attributeID}={value1}-{value2}. For date/ time values, we support an inclusive range on the tag. This will be mapped to `attributeID >= {value1} AND attributeID <= {value2}`. If {value1} is not specified, all occurrences of dates/times prior to and including {value2} will be matched. Likewise, if {value2} is not specified, all occurrences of {value1} and subsequent dates/times will be matched. However, one of these values has to be present. `{attributeID}={value1}-` and `{attributeID}=-{value2}` are valid, however, `{attributeID}=-` is invalid. |
+| Exact Match | All supported attributes | {attributeID}={value1} |
+| Fuzzy Match | PatientName, ReferringPhysicianName | Matches any component of the name which starts with the value. |
+
+#### Attribute ID
+
+Tags can be encoded in a number of ways for the query parameter. We have partially implemented the standard as defined in [PS3.18 6.7.1.1.1](http://dicom.nema.org/medical/dicom/2019a/output/chtml/part18/sect_6.7.html#sect_6.7.1.1.1). The following encodings for a tag are supported:
+
+| Value            | Example          |
+| :--------------- | :--------------- |
+| {group}{element} | 0020000D         |
+| {dicomKeyword}   | StudyInstanceUID |
+
+Example query searching for instances:
+`../instances?Modality=CT&00280011=512&includefield=00280010&limit=5&offset=0`
+
+### Search Response
+
+The response will be an array of DICOM datasets. Depending on the resource, by *default* the following attributes are returned:
+
+#### Default Study tags
+
+| Tag          | Attribute Name |
+| :----------- | :------------- |
+| (0008, 0020) | StudyDate |
+| (0008, 0050) | AccessionNumber |
+| (0008, 1030) | StudyDescription |
+| (0009, 0090) | ReferringPhysicianName |
+| (0010, 0010) | PatientName |
+| (0010, 0020) | PatientID |
+| (0010, 0030) | PatientBirthDate |
+| (0020, 000D) | StudyInstanceUID |
+
+#### Default Series tags
+
+| Tag          | Attribute Name |
+| :----------- | :------------- |
+| (0008, 0060) | Modality |
+| (0008, 1090) | ManufacturerModelName |
+| (0020, 000E) | SeriesInstanceUID |
+| (0040, 0244) | PerformedProcedureStepStartDate |
+
+#### Default Instance tags
+
+| Tag          | Attribute Name |
+| :----------- | :------------- |
+| (0008, 0018) | SOPInstanceUID |
+
+If `includefield=all`, below attributes are included along with default attributes. Along with default attributes, this is the full list of attributes supported at each resource level.
+
+#### Additional Study tags
```

Comment:
Above missing items appear to be moved here, which tells me that this was likely intentional, not an omission.  Is that right?